tasks/setup.js

```diff
@@ -377,22 +377,12 @@ module.exports = function (grunt) {
 
         var done        = this.async(),
```

Comment:
Now this is unused: remove or call it.
(I've had some trouble testing the PR because of this)